sentry.py

```
@@ -0,0 +1,101 @@
+
+print("Memory RSS: ", memory_rss)
+# print("CPU Percent: ", cpu_percent)
+print("CPU User: ", cpu_user)
+
+sentry_sdk.set_measurement("memory.rss", memory_rss, "byte")
+# sentry_sdk.set_measurement("cpu.percent", cpu_percent, "percent")
+sentry_sdk.set_measurement("cpu.user", cpu_user, "second")
+
+
+@sentry_sdk.trace
+def slow_function():
```

Ruff (0.8.2)

Missing return type annotation for public function `slow_function`

Add return type annotation: `str`

(ANN201)